packages/terra-date-input/src/DateInput.jsx

```diff
@@ -499,7 +510,7 @@ class DateInput extends React.Component {
           disabled={this.props.disabled}
           aria-describedby={`${this.formatDescriptionId} ${this.props.monthAttributes['aria-describedby']}`}
         >
-          <option value="" hidden>{this.props.intl.formatMessage({ id: 'Terra.date.input.monthPlaceholder' })}</option>
+          <option value="">{this.props.intl.formatMessage({ id: 'Terra.date.input.monthPlaceholder' })}</option>
```

Comment:
I thought placeholders were in the process of being removed? Should this still be hidden?

Reply:
This placeholder was not removed, and this is a fix for #1241